File: CTCI/ch1/URLify.swift
func URLify(inputString : String, stringSize : Int) -> String {
    var newString : String = "";
    for letter in inputString.characters {
        if(letter == " "){
            newString+="%20";
        } else {
            newString+=String(letter);
        }
        
    }
    return newString;
}

let string1 : String = "Mr John Smith";
print(URLify(inputString: string1, stringSize: 13));

/*
	> swift URLify.swift

	prints:
		> Mr%20John%20Smith
*/

File: CTCI/ch1/python/URLify.py
'''
	Write a method to replace all spaces in a string with '%20'. You can assume you are given the 'true'
	length of the string
'''

def URLify(inputString, length):
	return inputString.replace(' ', '%20')




string1 = 'Mr John Smith'
print(URLify(string1, 13))
'''
	prints:
		> Mr%20John%20Smith
'''